state_of_html.yml

```diff
@@ -898,7 +898,6 @@ translations:
 
   - key: features.features_intro_html
     t: |
-      Welcome to the survey! These first couple sections are all about figuring out
-      which **features** of HTML (and related APIs) you know about.
+      Bem-vindo à pesquisa! Estas primeiras seções são para descobrir quais **recursos** do HTML (e APIs relacionadas) você conhece.
 
-      By the way, if you want to learn more about any item in the survey, you can add it to your reading list by clicking its little "+" icon!
+      A propósito, se você quiser saber mais sobre qualquer item da pesquisa, você pode adicioná-lo à sua lista de leitura clicando no pequeno ícone "+"!
```

Comment:
Acho que seria mais fácil organizarmos as traduções por seções, e esse trecho aqui não faz parte da seção de Forms. Apenas uma sugestão pra mantermos o contexto e facilitar na hora de revisar.

Reply:
Erro meu, não coloquei o nome corretamente na PR, mas tinha já organizado para traduzir as partes mencionada em https://github.com/Devographics/locale-pt-BR/issues/71, só não subi das sections porque foi feito um outro PR com esta parte antes dessa PR em si

Reply:
Tranqulo. Caso tenha interesse em traduzir outras seções, sugiro fazer uma por vez.

Podemos organizar isso por issue também, assim evitamos mais de um traduzir a mesma seção.

Reply:
Acho que a _issue_ completa pode permanecer, @romulo1984. Mas a recomendação é realmente traduzir seções isoladas dentro do arquivo, assim contribuidores diferentes podem traduzir o mesmo arquivo simultaneamente.

Vi que vocês já estão usando o [**Projects**](https://github.com/Devographics/locale-pt-BR/projects/1) e tá muito bom.

@andreseichi só recomendo você escolher uma seção por vez. Por exemplo, em vez de colocar `state_of_html.yml` lá no _kanban_ tu especifica a seção que vai traduzir: `state_of_html.yml - Forms`, dai todo mundo sabe que você tá traduzindo a seção `Forms` dentro do arquivo `state_of_html.yml`.

Reply:
@the-physicist e @romulo1984 vlw pelas observações.
Irei organizar melhor para futuras traduções no projeto